### DISCUSSION TITLE
What Storybook Plugins should we look into adding?

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/open-sauced#1057] [Ideas]
With the following PRs pending #1037 #1055, it may be interesting to think of which storybook plugins would be valuable to add. 

So far #1053 was mentioned. 

List ideas or link examples you think would be interesting to look into. 

Here is a tweet with quite a few ideas. https://twitter.com/bdougieYO/status/1403019695431716865

**Comment by 0-vortex** [comment]
None of these links are forked or tested locally, but these are all the storybook add-ons that 👀 interesting to implement for open-sauced:
- [atlassian-labs/storybook-addon-performance](https://github.com/atlassian-labs/storybook-addon-performance) could help us figure out bottlenecks
- [UX-and-I/storybook-design-token](https://github.com/UX-and-I/storybook-design-token) could help us standardise missing and wireframe designs similar to theming frameworks like theme-ui
- [pocka/storybook-addon-designs](https://github.com/pocka/storybook-addon-designs) could help us embed original component design as simple figma links
- [tuchk4/storybook-readme](https://github.com/tuchk4/storybook-readme) not sure this even works anymore but the idea of sticking to GFM is pretty solid
